addons/sale/views/product_views.xml

```diff
@@ -95,8 +95,11 @@
         <field name="name">Products</field>
         <field name="res_model">product.template</field>
         <field name="path">products</field>
-        <field name="view_mode">kanban,list,form,activity</field>
-        <field name="view_id" ref="product.product_template_kanban_view"/>
+        <field name="view_ids" eval="[Command.clear(),
+            Command.create({'view_mode': 'kanban'}),
+            Command.create({'view_mode': 'list', 'view_id': ref('account.product_template_list_view_sellable_inherit')}),
```

Comment:
why do you reference that specific `account` view ?
It's an inherited view, you should reference the primary parent if really needed, or let it fall back on the default one.